packages/widget/src/components/resource-amount-selector/resource-amount-selector.ts

```diff
@@ -163,14 +163,23 @@ export class ResourceAmountSelector extends BaseComponent {
   }
 
   _normalizeOptions(): DropdownOption<Resource>[] {
-    return when(this.resources, () =>
-      this.resources.map((entry) => ({
+    return when(this.resources, () => {
+      const resources = this.resources.map((entry) => ({
         id: entry.resourceId,
         name: entry.symbol!,
         icon: networkIconsMap.default,
         value: entry
-      }))
-    );
+      }));
+
+      resources.unshift({
```

Comment:
this adds "select token" as an option to be selected, which might be confusing

Reply:
got it, added placeholder option